### PR TITLE
feat: social buttons css change

### PR DESCRIPTION
**Pull request by alisher-aituarov**
## Summary of change

Changed stylings for social auth buttons

## Related issues
https://github.com/supertokens/main-website/issues/977

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

## Documentation changes

(If relevant, please create a PR in our [docs repo](https://github.com/supertokens/docs), or create a checklist here highlighting the necessary changes)

## Checklist for important updates

-   [x] Changelog has been updated
-   [x] `frontendDriverInterfaceSupported.json` file has been updated (if needed)
-   [x] Changes to the version if needed
    -   In `package.json`
    -   In `package-lock.json`
    -   In `lib/ts/version.ts`
-   [x] Had run `npm run build-pretty`
-   [x] Had installed and ran the pre-commit hook
-   [x] Issue this PR against the latest non released version branch.
    -   To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
    -   If no such branch exists, then create one from the latest released branch.
-   [ ] If added a new recipe interface, then make sure that the implementation of it uses NON arrow functions only (like `someFunc: function () {..}`).
-   [ ] If I added a new recipe, I also added the recipe entry point into the `size-limit` section of `package.json` with the size limit set to the current size rounded up.

## Remaining TODOs for this PR

-   [ ]

### CODE REVIEW

File: lib/ts/recipe/thirdparty/components/themes/styles.ts [hunk 1]
```diff
@@ -40,46 +40,53 @@ export function getStyles(palette: NormalisedPalette): NormalisedDefaultStyles {
...
         },
 
         providerButtonLogoCenter: {
+            display: "flex",
             margin: "auto",
         },
 
         providerButtonText: {
-            margin: "auto",
+            fontFamily: "Rubik",
```
Comment (porcellus) [inline lib/ts/recipe/thirdparty/components/themes/styles.ts:76]: I don't think we need to set this here

File: lib/ts/recipe/thirdparty/components/themes/styles.ts [hunk 2]
```diff
@@ -18,12 +18,12 @@ import { getButtonStyle, getDefaultStyles, getMergedStyles } from "../../../../s
 import { NormalisedPalette, NormalisedDefaultStyles } from "../../../../types";
 
 const providerColors = {
```
Comment (porcellus) [inline lib/ts/recipe/thirdparty/components/themes/styles.ts:20]: I don't think we need this const anymore.

File: lib/ts/recipe/thirdparty/components/themes/styles.ts [hunk 3]
```diff
@@ -40,46 +40,53 @@ export function getStyles(palette: NormalisedPalette): NormalisedDefaultStyles {
...
             textAlign: "center",
             justifyContent: "center",
         },
 
-        providerGoogle: getButtonStyle(providerColors.google, "white"),
-        providerGitHub: getButtonStyle(providerColors.github, "white", true),
-        providerTwitter: getButtonStyle(providerColors.twitter, "white"),
-        providerFacebook: getButtonStyle(providerColors.facebook, "white"),
-        providerApple: getButtonStyle(providerColors.apple, "white", true),
+        providerGoogle: getButtonStyle(providerColors.google, "black"),
```
Comment (porcellus) [inline lib/ts/recipe/thirdparty/components/themes/styles.ts:82]: We can still keep these styles separate as an option for customization.

**Review by porcellus — CHANGES_REQUESTED** [review]

File: lib/ts/recipe/thirdparty/components/themes/styles.ts (hunk 1 above)
Comment (alisher-aituarov) [reply lib/ts/recipe/thirdparty/components/themes/styles.ts:76]: do you want me to set it somewhere else?

File: lib/ts/recipe/thirdparty/components/themes/styles.ts (hunk 3 above)
Comment (alisher-aituarov) [reply lib/ts/recipe/thirdparty/components/themes/styles.ts:82]: something like 
`const providerTextColors = { /*...*/ }` ?

File: lib/ts/recipe/thirdparty/components/themes/styles.ts (hunk 1 above)
Comment (porcellus) [reply lib/ts/recipe/thirdparty/components/themes/styles.ts:76]: We are using Rubik by default everywhere, doesn't it get applied here?

File: lib/ts/recipe/thirdparty/components/themes/styles.ts (hunk 3 above)
Comment (porcellus) [reply lib/ts/recipe/thirdparty/components/themes/styles.ts:82]: I'd prefer if you added something like `const defaultProviderButtonStyle = getButton...` and added that to props here.

**Review by porcellus — CHANGES_REQUESTED** [review]
- the styles are not right in thirdpartyemailpassword (the button texts are not centered)
- the divider should be removed from between the headers and the provider list
- the facebook logo is different
- the provider buttons in the third-party design are wider than what the "row" class allows. This might not be intentional since we are using the row class everywhere else. So I think this should stay as-is for now (for consistency), and we should discuss this with @mufassirkazi.

**Review by porcellus — APPROVED** [review]

**Review by porcellus — CHANGES_REQUESTED** [review]
thirdpartypasswordless is looking different (it should match the look of thirdpartyemailpassword)

**Comment by porcellus** [comment]
I'll paste the facebook logo svg here, hopefully it uploads right. Otherwise, you can download it from here and convert from eps to SVG: https://www.facebook.com/brand/resources/facebookapp/logo
f_logo_RGB-Blue_1024

**Comment by alisher-aituarov** [comment]
> thirdpartypasswordless is looking different (it should match the look of thirdpartyemailpassword)

everything looks good after deleting .`cache` from examples `node_modules`

**Review by porcellus — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [0.27.5] - 2022-12-08
```
Comment (rishabhpoddar) [inline CHANGELOG.md:10]:
```suggestion
```

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: lib/ts/recipe/thirdparty/components/themes/styles.ts [hunk 5]
```diff
@@ -17,14 +17,7 @@ import chroma from "chroma-js";
 import { getButtonStyle, getDefaultStyles, getMergedStyles } from "../../../../styles/styles";
 import { NormalisedPalette, NormalisedDefaultStyles } from "../../../../types";
 
-const providerColors = {
-    google: "#ea3721",
-    github: "#000",
-    facebook: "#274483",
-    twitter: "#008dd1",
-    apple: "#07093c",
-    custom: "#FFF",
```
Comment (rishabhpoddar) [inline lib/ts/recipe/thirdparty/components/themes/styles.ts:26]: please redo how custom provider's default UI button shows -> the alignment of the text in the button is a bit off

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+### Changed
```
Comment (rishabhpoddar) [inline CHANGELOG.md:10]:
```suggestion
### Breaking change
```

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

**Comment by rishabhpoddar** [comment]
@mufassirkazi already released the change we had discussed yest.

**Pull request merged**